### PR TITLE
[SMALLFIX]Append -raw option to show the file's raw size

### PR DESCRIPTION
**Pull request by maobaolong**
Default in byte size.This PR's purpose is that some user want to see the file byte size,

### CODE REVIEW

File: shell/src/main/java/alluxio/shell/command/LsCommand.java [hunk 1]
```diff
@@ -64,12 +66,18 @@ public static String formatLsString(boolean acl, boolean isFolder, String permis
     } else {
       memoryState = inMemory ? STATE_FILE_IN_MEMORY : STATE_FILE_NOT_IN_MEMORY;
     }
+    String sizeStr = "";
```
Comment (yupeng9) [inline shell/src/main/java/alluxio/shell/command/LsCommand.java]: nit: You can use String sizeStr = humanReadable ? FormatUtils.getSizeFromBytes(size):String.valueOf(size);

Comment (maobaolong) [inline shell/src/main/java/alluxio/shell/command/LsCommand.java]: Get it. `Ternary Operator` is the wiser way, Thank you.

**Comment by maobaolong** [comment]
@yupeng9  PTAL.

File: shell/src/main/java/alluxio/shell/command/LsCommand.java [hunk 2]
```diff
@@ -55,7 +56,8 @@
    * @param path path of the file or folder
    * @return the formatted string according to acl and isFolder
    */
-  public static String formatLsString(boolean acl, boolean isFolder, String permission,
+  public static String formatLsString(boolean humanReadable, boolean acl, boolean isFolder, String
```
Comment (aaudiber) [inline shell/src/main/java/alluxio/shell/command/LsCommand.java]: please reformat these arguments

File: shell/src/main/java/alluxio/shell/command/LsCommand.java [hunk 3]
```diff
@@ -162,19 +170,21 @@ public int compare(URIStatus status1, URIStatus status2) {
 
   @Override
   public void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
-    ls(path, cl.hasOption("R"), cl.hasOption("f"), cl.hasOption("d"));
+    ls(path, cl.hasOption("R"), cl.hasOption("f"), cl.hasOption("d"),
+        cl.hasOption("h"));
```
Comment (aaudiber) [inline shell/src/main/java/alluxio/shell/command/LsCommand.java]: does this need to be wrapped?

**Comment by maobaolong** [comment]
@aaudiber ok, i will use  -raw argument, but what do you think if a single  letter flag argument is better?

**Comment by aaudiber** [comment]
I think `-raw` is a lot more readable than using an arbitrary single letter. `R` is already taken by recursive.

**Comment by maobaolong** [comment]
@aaudiber okey, let me start a modification.

File: shell/src/main/java/alluxio/shell/command/LsCommand.java (hunk 3 above)
Comment (maobaolong) [inline shell/src/main/java/alluxio/shell/command/LsCommand.java]: No, it is a wrong wrap. Now I have been corrected it.

File: shell/src/main/java/alluxio/shell/command/LsCommand.java (hunk 2 above)
Comment (maobaolong) [inline shell/src/main/java/alluxio/shell/command/LsCommand.java]: I have changed the argument to `raw`

**Comment by maobaolong** [comment]
@aaudiber  PTAL.

**Comment by yupeng9** [comment]
I actually prefer `ls` to be more like of Linux, with a `-h` option so people do not need any learning of this command. @calvinjia what do you think?

**Comment by maobaolong** [comment]
@yupeng9 actually, I support your viewpoint, i use the `h` flag just think about linux command `ls`.

So let us discuss and make a decision?

@aaudiber , @calvinjia

**Comment by aaudiber** [comment]
Unlike a typical Unix filesystem, files in Alluxio tend to be much larger, such that printing raw bytes makes it difficult for a human to read. We're already departing from typical shells by giving file size even without the `-l` option. Our shell is already very different from standard shell builtins, so I think we should choose defaults which make more sense for a human inspecting the filesystem, which is the common use case

**Comment by calvinjia** [comment]
I think the default should be printed in -h mode for the reasons @aaudiber mentioned.

**Comment by maobaolong** [comment]
Both your opinions are all make sense,  It doesn't matter for me, now it is -h default, the flag -raw to show byte size. If you make decision, please merge this.

**Pull request merged**